hedera-mirror-test/traffic-replay/log-converter/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "log-converter",
```

Even if it's private, we don't want a hacker one report about a npm takeover.

```suggestion
  "name": "@hashgraph/mirror-log-converter",
```